omod/src/main/webapp/fragments/patientHeader.gsp

```diff
@@ -81,6 +81,9 @@
             </span>
             <span class="death-info-extension">
                 <%= ui.includeFragment("appui", "extensionPoint", [ id: "patientHeader.deathInfo", contextModel: appContextModel ]) %>
+                <% if (context.hasPrivilege("Task: Edit Vital Status")) { %>
```

Comment:
Shouldn't you instead be checking for the "Ability to Mark a patient dead/deceased" privilege?

Reply:
I see that you added two new privileges in the config.xml file. Does this render the second one redundant?

Reply:
Removed one privilege which was redundant